repos/spack_repo/builtin/packages/darshan_util/package.py

```diff
@@ -77,6 +78,12 @@ class DarshanUtil(AutotoolsPackage):
 
     patch("retvoid.patch", when="@3.2.0:3.2.1")
 
+    def url_for_version(self, version):
+        if version < Version("3.5.0"):
+            return f"https://web.cels.anl.gov/projects/darshan/releases/darshan-{version}.tar.gz"
+        else:
+            return f"https://github.com/darshan-hpc/darshan/archive/refs/tags/{version}.tar.gz"
```

Comment:
```suggestion
    def url_for_version(self, version):
        if self.spec.satisfies("@:3.4"):
            return f"https://web.cels.anl.gov/projects/darshan/releases/darshan-{version}.tar.gz"
        else:
            return f"https://github.com/darshan-hpc/darshan/archive/refs/tags/{version}.tar.gz"
```

Same suggestion here.